dev/com.ibm.ws.app.manager.wab/src/com/ibm/ws/app/manager/wab/internal/WABInstaller.java

```diff
@@ -288,7 +288,7 @@ protected void activate(ComponentContext context) {
 
         tracker = getTracker(customizer);
         tracker.open();
-        wabLifecycleDebug("Master WAB Tracker has opened.");
+        wabLifecycleDebug("Primary WAB Tracker has opened.");
```

Comment:
@tjwatson can you confirm that there's no concerns regarding changing these debug statements?

Reply:
These are only debug messages that end up in trace.log if enabled.  We must be able to change this with no issue.

Reply:
@anjumfatima90 just keep an eye out on tests in this project in the PB in case they were looking for this string value (though I'm guessing you also searched for 'master' in tests so you would have found it; aka, very unlikely tests would be impacted)